inference-engine/src/mkldnn_plugin/mkldnn_node.cpp

```diff
@@ -732,6 +732,54 @@ void MKLDNNNode::initDescriptor(const InferenceEngine::LayerConfig &config) {
     selectedPD->getConfig() = rightConfig;
 }
 
+InferenceEngine::Blob::Ptr MKLDNNNode::createInternalBlob(InferenceEngine::SizeVector dims, size_t edgeNum, bool isGrouped) {
```

Comment:
Since it is used only for Convolutin I would prefer to put it as private Convolution method. Moreover it should be simplified I suppose.

Reply:
done